tests/unit/disc/omegah/box_mesh.cpp

```diff
@@ -0,0 +1,36 @@
+//*****************************************************************//
+//    Albany 3.0:  Copyright 2016 Sandia Corporation               //
+//    This Software is released under the BSD license detailed     //
+//    in the file "license.txt" in the top-level Albany directory  //
+//*****************************************************************//
+
+#include "Albany_OmegahBoxMesh.hpp"
+#include "Albany_CommUtils.hpp"
+#include "Albany_Utils.hpp"
+
+#include "Teuchos_CommHelpers.hpp"
+#include "Teuchos_UnitTestHelpers.hpp"
+#include "Teuchos_LocalTestingHelpers.hpp"
+
+TEUCHOS_UNIT_TEST(OmegahBoxMesh, 2D)
+{
+  using namespace Albany;
+
+  build_type (BuildType::Tpetra);
+  auto comm = getDefaultComm();
+
+  const int nelemx = 4;
+  const int nelemy = 3;
+  const int nelemz = 2;
+  const int num_params = 2;
+
+  auto params = Teuchos::rcp(new Teuchos::ParameterList(""));
+  params->set<int>("1D Elements",nelemx);
+  params->set<int>("2D Elements",nelemy);
+  params->set<int>("3D Elements",nelemz);
```

Comment:
If it wouldn't require significant work/fiddling `[D1|D2|D3] Elements` (or if we don't care about string length ... `Number of Elements: [X|Y|Z]`) would be a bit clearer than `[1D|2D|3D] Elements`.

Reply:
I agree. I guess I stuck with the same syntax used for the STK version, but it's not a great syntax, so I'm happy to change it.